bash_it.sh

```diff
@@ -121,5 +121,12 @@ if ! command -v reload &>/dev/null && [ -n "$BASH_IT_RELOAD_LEGACY" ]; then
   esac
 fi
 
+# If gitstatusd is requested for the current shell, launch it if the shell is interactive
```

Comment:
I'd prefer to have this in a new plugin. Since it adds optional functionality, please move it to its own `gitstatusd` plugin.

Reply:
I see your point, question would be if this shouldn't become the new standard some time in the future as the performance increase is very significant.

Reply:
@markusdd speaking only for myself, the issue is never performance, it's opinionatation. You're forcing your opinion on others. I for one would don't care how efficient `git status` is, because I don't work on repos so big that it matters. I would bet that is the case for 99% of users. I would encourage you to use a plugin instead.

Reply:
Yes, that's how I see it as well. This really should be in its own plugin as an option. Personally, I don't have the issue with large repos (not enough that it's bothering me), and having to install an external dependency is a big ask...

Reply:
That is fine by me, problem is that I do not have experience with bash-it plugins and my time is severly limited right now unfortunately. Is anyone able to take over moving the launch into a plugin?

Reply:
Sorry, I won't have time for this.

Reply:
See #1622
Moved into a plugin :smile: